frontend/api.py

```diff
@@ -371,8 +371,9 @@ def api_volumes():
 		comicvine_id = extract_key(request, 'comicvine_id')
 		root_folder_id = extract_key(request, 'root_folder_id')
 		monitor = extract_key(request, 'monitor', False)
+		issues_as_volumes = extract_key(request, 'issues_as_volumes', False)
```

Comment:
By giving `False` for the third argument, you say that it's not required to supply this parameter. However, no default value is set for the parameter in the case that it is not supplied. Default values should be set in `api.py` around line 143.